RecoCTPPS/TotemRPLocal/python/totemRPLocalReconstruction_cff.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+# geometry
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+XMLIdealGeometryESSource_CTPPS.geomXMLFiles.append("Geometry/VeryForwardData/data/2016_ctpps_15sigma_margin0/RP_Dist_Beam_Cent.xml")
```

Comment:
looking at this again, after noticing that the XML file has changed as well,
I think it makes more sense to define several instances of XMLIdealGeometryESSource
XMLIdealGeometryESSource_CTPPS_15sigma XMLIdealGeometryESSource_CTPPS_garage etc in geometryRP_cfi (unique place) and then pick up the one you want with a config parameter in reconstruction.

This will prevent possibility of accidental multiple geomXMLFiles.append
(imagine you need it also in some code that doesn't also run totemRPLocalReconstruction,
then depending on that code loaded together or separately with totemRPLocalReconstruction you may get different content of geomXMLFiles.append, worse yet it can depend on the order these files are loaded)